pkg/util/pravegacluster.go

```diff
@@ -33,6 +33,10 @@ func init() {
 	versionRegexp = regexp.MustCompile(MajorMinorVersionRegexp)
 }
 
+func ConfigMapNameForOperatorVersions() string {
```

Comment:
This is not a config map for "operator versions" It is a map for "pravega versions" Please provide a suitable name....